tests/plugins/test_flatpak_create_oci.py

```diff
@@ -67,9 +67,9 @@
 
 ICON = BytesIO()
 # create minimal 256x256 RGBA PNG
-png.Writer(256, 256, alpha=True).write(ICON,
-                                       [[0 for _ in range(4 * 256)]
-                                        for _ in range(256)])
+png.Writer(256, 256, greyscale=False, alpha=True).write(ICON,
```

Comment:
Does this fix the issue with the new pypng? If so, could you also remove the freeze on pypng 0.0.19 in tests/requirements.txt?

Reply:
Done